modules/gltf/doc_classes/GLTFTexture.xml

```diff
@@ -9,7 +9,7 @@
 	<methods>
 	</methods>
 	<members>
-		<member name="src_image" type="int" setter="set_src_image" getter="get_src_image" default="212600976">
+		<member name="src_image" type="int" setter="set_src_image" getter="get_src_image" default="0">
```

Comment:
Don't include unrelated changes caused by doctool in the commit. (Use `git add -p` and `git reset -p` to make partial commits.)

Reply:
Thanks for the quick turnaround. I included these changes because that's how I understood the [Updating the documentation template doc here](https://docs.godotengine.org/en/stable/community/contributing/updating_the_class_reference.html#updating-the-documentation-template):

> If there are changes to other classes than the one you are planning to document, please commit those changes first before starting to edit the template: